src/Data/Set.purs

```diff
@@ -176,3 +177,11 @@ intersection s1 s2 = fromFoldable $ runPure (runSTArray (emptySTArray >>= inters
         LT -> pure $ Loop {a: l + 1, b: r}
         GT -> pure $ Loop {a: l, b: r + 1}
       else pure $ Done acc
+
+-- | The set of all subsets of specified size
```

Comment:
I think the following are missing:

* Running time, preferably in big O
* An example in the docs
* At least one full sentence in the docs - this seems a little unnecessarily abridged. In particular, what happens when the provided `Int` argument is larger than the size of the provided set?
* Handling for the case where a negative `Int` argument is supplied.

Reply:
Yes, you're absolutely right. I threw both this and `powerSet` over the fence a bit prematurely. I'll fix them up.